sdks/java/io/cassandra/src/test/java/org/apache/beam/sdk/io/cassandra/CassandraIOTest.java

```diff
@@ -268,6 +267,36 @@ public void testWrite() {
     }
   }
 
+  @Test
+  public void testWriteWithPasswordDecryption() {
```

Comment:
please also add a testReadWithPasswordDecryption as this feature is also usable at read time. I agree it should work but it is worth testing it (Source is different than Write DoFn).